### PR TITLE
fix: enable benchmark

### PR DESCRIPTION
**Pull request by tedxu**

### CODE REVIEW

**Comment by tedxu** [comment]
Benchmark result:

```
------------------------------------------------------------------------
Benchmark                              Time             CPU   Iterations
------------------------------------------------------------------------
S3Fixture/Write32MB/real_time   43236907 ns     38985846 ns           13
S3Fixture/Read32MB/real_time     9751422 ns      8241057 ns           70
```

Write is much slower than read, not sure if it is expected.

File: cpp/benchmark/benchmark_packed.cpp [hunk 1]
```diff
@@ -75,22 +78,23 @@ static void PackedRead(benchmark::State& st, arrow::fs::FileSystem* fs, const st
...
-    auto r = pr.ToTable();
-    if (!r.ok()) {
-      st.SkipWithError(r.status().ToString());
-    }
-    auto table = r.ValueOrDie();
-    r = pr.Close();
-    if (!r.ok()) {
-      st.SkipWithError(r.status().ToString());
+    auto r = arrow::RecordBatch::MakeEmpty(schema);
+    SKIP_IF_NOT_OK(r.status(), st)
```
Comment (shaoting-huang) [inline cpp/benchmark/benchmark_packed.cpp:90]: Does it affect the benchmark result if we skip?

**Review by shaoting-huang — APPROVED** [review]

File: cpp/benchmark/benchmark_packed.cpp (hunk 1 above)
Comment (tedxu) [reply cpp/benchmark/benchmark_packed.cpp:90]: Skip means abort the benchmark if there is error.

**Pull request merged**